controllers/complianceeventsapi/server.go

```diff
@@ -180,6 +181,18 @@ func (s *complianceAPIServer) Start(dbURL string) error {
 		getSingleComplianceEvent(db, w, r)
 	})
 
+	mux.HandleFunc("/api/v1/reports/compliance-events", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
You aren't returning JSON. This can be removed since `getComplianceEventsCSV` sets the correct header.

Reply:
This for err, I am setting again near csv writing